### PR TITLE
feat(events-v2): Support ordering by a list of fields

### PR DESCRIPTION
**Pull request by lynnagara**
We want to be able to sort events by multiple fields e.g. `["-timestamp", "-id"]`

### CODE REVIEW

File: src/sentry/utils/snuba.py [hunk 1]
```diff
@@ -264,7 +264,7 @@ def zerofill(data, start, end, rollup, orderby):
         else:
             rv.append({'time': key})
 
-    if orderby.startswith('-'):
+    if len(orderby) > 0 and orderby[0].startswith('-'):
```
Comment (markstory) [inline src/sentry/utils/snuba.py]: How does this interact with multiple sort conditions when the second is not prefixed with `-`?

Comment (lynnagara) [reply src/sentry/utils/snuba.py]: @billyvg Do you know how what the desired behavior here should be? At a glance it seems to me like this is always sorting by time regardless of the orderby column specified?

Comment (billyvg) [reply src/sentry/utils/snuba.py]: Yeah it seems like it incorrectly assumes orderby is always `time` - we would look for `time` in `orderby` and figure out if it's asc/desc

Comment (lynnagara) [reply src/sentry/utils/snuba.py]: Sounds good

Comment (lynnagara) [reply src/sentry/utils/snuba.py]: Updated, thanks for catching this!

**Review by markstory — APPROVED** [review]

**Pull request merged**